style.css

```diff
@@ -1,4 +1,347 @@
+        display: flex;
+        justify-content: space-between;
+    }
+    
+    .popup1size {
+        width: 290px;
+        height: 687px;
+    }
+    
+    .popup2size {
+        width: 290px;
+        height: 280px;
+    }
+    
+    .list {
+        display: flex;
+        flex-wrap: wrap;
+    }
+    
+    .list li {
+        list-style-type: none;
+    }
+    
+    .list li a {
+        display: inline-block;
+        border: 1px solid transparent;
+        background: #fff;
+        padding: 3px 21px;
+        margin-bottom: 10px;
+        margin-right: 6px;
+        transition: all 0.3s ease 0s;
+        color: #777777;
+        font-size: 14px;
+        font-family: 'Source Sans Pro', sans-serif;
+        text-decoration: none;
+    }
+    
+    .list li a:hover {
+        background: #222222;
+        color: #ff9907 !important;
+    }
+    
+    .boxcont4size {
+        width: 220px;
+        height: 200px;
+    }
+    
+    .thumb {
+        position: relative;
+        width: 230px;
+        height: 106px;
+    }
+    
+    .thumbinfo {
+        position: absolute;
+        left: 0px;
+        bottom: 0px;
+        background-color: #fff;
+    }
+    
+    .thumbinfo ul li,
+    .imgc ul li {
+        display: inline-block;
+        list-style-type: none;
+        padding: 0px 10px 5px;
+    }
+    
+    .thumbinfo a,
+    .imginfo a {
+        text-decoration: none;
+        font-size: 14px;
+        font-family: 'Source Sans Pro', sans-serif;
+        color: #797979;
+    }
+    
+    .popup1 h6 {
+        transition: all 0.3s ease 0s;
+        color: coral;
+    }
+    
+    .popup1 h6:hover {
+        color: #ff9907 !important;
+    }
+    
+    .imginfo li {
+        padding: 15px 0 !important;
+        margin-right: 22px;
+    }
+    
+    .imginfo a {
+        font-size: 15px;
+        font-family: 'Source Sans Pro', sans-serif;
+        ;
+    }
+    
+    .imginfo ul {
+        padding-left: 20px;
+    }
+    
+    .clr {
+        color: #999;
+    }
+    
+    .blog1 {
+        width: 610px;
+        height: 100%;
+    }
+    
+    .bbtnw {
+        width: 170px;
+        height: 47px;
+    }
+    
+    #pagemidcont {
+        font-family: 'Source Sans Pro', sans-serif;
+    }
+    
+    #pagemidcont h3 {
+        font-size: 27px;
+        font-family: 'Lora', serif;
+        color: #3A414E;
+    }
+    
+    #pagemidcont a {
+        text-decoration: none;
+        color: #222;
+    }
+    
+    #pagemidcont h6 {
+        font-size: 16px;
+        font-family: 'Lora', serif;
+        color: #3A414E;
+    }
+    
+    .imgc {
+        position: relative;
+        width: 610px;
+        height: 325px;
+    }
+    
+    .imginfo {
+        width: 415px;
+        height: 54px;
+        position: absolute;
+        left: 0px;
+        bottom: 0px;
+        background-color: #fff;
+    }
+    /*
+    @media (min-width: 768px) and (max-width: 1024px) {}
+    @media (min-width: 481px) and (max-width: 767px) {}
+    610*/
```

Comment:
remove the comments here, since you don't need them

Reply:
okay